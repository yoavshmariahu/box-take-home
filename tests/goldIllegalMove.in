k a1
K e5
g b2
G d4

[S B R P]
[p r b s]

move b2 b1
move d4 c5
