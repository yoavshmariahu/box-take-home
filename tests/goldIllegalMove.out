UPPER player action: move d4 c5
5 |__|__|__|__| K|
4 |__|__|__| G|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k| g|__|__|__|
    a  b  c  d  e

Captures UPPER: S B R P
Captures lower: p r b s

lower player wins.  Illegal move.
